bindings/python/tools/python/ostk/astrodynamics/__init__.py

```diff
@@ -3,3 +3,9 @@
 from ostk.physics import *
 
 from .OpenSpaceToolkitAstrodynamicsPy import *
+
+from .pytrajectory.pystate import State as PyState
```

Comment:
Curious if there are any better ideas here. This works, but it means State is imported whenever anything else in Astro is imported. State is already very pervasive, so it may not make much difference, but it's still messy.